### DISCUSSION TITLE
Is firecfg supposed to also restore the .desktop files in user's HOME folder?

### DISCUSSION THREAD

**Discussion by mystiquewolf** [netblue30/firejail#5016] [Q&A] [answered]
I've used `sudo firecfg`, it added its symlinks to `/usr/local/bin/` and also converted the `.desktop` files in my home folder. However neither running `sudo firecfg --clean` nor simply `firecfg --clean` restores the symlinks in my `.local/share/applications/org.kde.spectacle.desktop` folder... How to restore them?

**Comment by rusty-snake** [comment, marked as answer]
`--clean` only cleans the symlinks. Other things like `--add-user` or `--fix` must be reverted manually.

**Comment by mystiquewolf** [comment]
Well that's not good at all. Firejail should make backups and restore the things it changes. I propose this to be done somehow.
